sections/dir.zsh

```diff
@@ -25,7 +25,7 @@ spaceship_dir() {
 
   # Threat repo root as a top-level directory or not
   if [[ $SPACESHIP_DIR_TRUNC_REPO == true ]] && spaceship::is_git; then
-    local git_root=$(git rev-parse --show-toplevel)
+    local git_root=${$(git rev-parse --absolute-git-dir):h}
```

Comment:
This requires at least `git v2.13.0`. Because `--absolute-git-dir` [was added with that release.](https://github.com/git/git/commit/a2f5a8762693d5af6dbbca714d882a21d7ba0b75)